activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Allow associations supporting the `dependent:` key to take `dependent: :destroy_later`.
+
+    ```ruby
+    class Account < ActiveRecord::Base
+        belongs_to :supplier, dependent: :destroy_later
+    end
+    ```
+
+    `:destroy_later` will enqueue a job to destroy associated records via `ActiveRecord::Base.destroy_later`.
```

Comment:
`ActiveRecord::Base.destroy_later` doesn't exist anymore. I think we can just say
```suggestion
    `:destroy_later` will enqueue a job to destroy associated records in background.
```